README.md

```diff
@@ -1,6 +1,12 @@
 # Hello Microverse
```

Comment:
- [ ] Branch should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), kindly consider following this rule in your future projects.
  #### you can skip this suggestion for this project,